tests/evergreen/test_version.py

```diff
@@ -8,7 +8,7 @@
 
 from evergreen.manifest import Manifest
 from evergreen.metrics.versionmetrics import VersionMetrics
-from evergreen.version import Requester, Version
+from evergreen.version import RecentVersions, Requester, Version
```

Comment:
We are testing `RecentVersions` which encompasses & indirectly tests `RecentVersionRow`. Would it be meaningful to add explicit tests for `RecentVersionRow`?